mssql/operations.py

```diff
@@ -188,6 +188,24 @@ def datetime_trunc_sql(self, lookup_type, field_name, tzname):
             sql = "CONVERT(datetime2, CONVERT(varchar, %s, 20))" % field_name
         return sql
 
+    def fetch_returned_insert_rows(self, cursor):
+        """
+        Given a cursor object that has just performed an INSERT...
```

Comment:
Very minor, but to mirror the PostgreSQL version of that comment (INSERT...RETURNING) we could go with this suggestion
```suggestion
        Given a cursor object that has just performed an INSERT...OUTPUT INSERTED
```